src/map/mob.c

```diff
@@ -3515,8 +3515,6 @@ static int mobskill_use(struct mob_data *md, int64 tick, int event)
 					flag = ((fmd = mob->getfriendstatus(md, ms[i].cond1, ms[i].cond2)) != NULL); break;
 				case MSC_SLAVELT: // slave < num
 					flag = (mob->countslave(&md->bl) < c2 ); break;
-				case MSC_ATTACKPCGT: // attack pc > num
-					flag = (unit->counttargeted(&md->bl) > c2); break;
```

Comment:
it's not same. comparison ``>`` and ``>=``
probably if this things going from aegis, better left all existing constants and code?

Reply:
You're right. It's not the same. But `MSC_ATTACKPCGT` is not used and `MSC_ATTACKPCGE` (which is used) can do the same, you just have to increase the value for `ConditionData` by 1.

Reply:
yes, but i mean if it was added because exists in aegis, probably better left it as is?

Reply:
@Kenpachi2k13 it's not used in our default database, people might be using it in theirs you can't just remove it without a reason..

Reply:
@Asheraf 
I understand your point of view, but nonetheless I think it's okay to remove those two constants.

- They are not documented so nobody should know about them.
- The feature itself isn't removed. One just need to use another constant and increase the value for `ConditionData` by 1.
- Following your argumentation it would be "forbidden" to rename constants, too, even though their names are misleading.
- Having two constants of same purpose is pointless.